op-e2e/system_tob_test.go

```diff
@@ -10,8 +10,8 @@ import (
 	"testing"
 	"time"
 
-	legacybindings "github.com/ethereum-optimism/optimism/op-bindings/bindings"
 	"github.com/ethereum-optimism/optimism/op-chain-ops/crossdomain"
+	legacybindings "github.com/ethereum-optimism/optimism/op-e2e/bindings"
```

Comment:
This should be updated here or in a follow up pr to be `bindings` instead of `legacybindings` now that these are canonical

Reply:
Cannot remove all references due to some type issues between packages